autorag/schema/metricinput.py

```diff
@@ -0,0 +1,100 @@
+from dataclasses import dataclass
+from typing import Optional, List, Dict, Callable, Any, Union
+
+import numpy as np
+import pandas as pd
+
+
+@dataclass
+class MetricInput:
+    query: Optional[str] = None
+    queries: Optional[List[str]] = None
+    gt_contents: Optional[List[str]] = None
+    retrieval_contents: Optional[List[str]] = None
```

Comment:
I think retrieval_contents should be List[List[str]] like retrieval_gt?

Reply:
Plus what is different of `gt_contents` and `retrieval_contents`?
I think it should be `retrieval_gt_contents` and `retrieved_contents`.
`retrieval_gt_contents` => The contents of the retrieval ground truth
`retrieved_contents` => The contents that retrieved by retrieval, reranker, or passage augmenter node.